src/ToricVarieties/NormalToricVarieties/constructors.jl

```diff
@@ -119,6 +119,21 @@ end
 
 export NormalToricVariety
 
+
+@doc Markdown.doc"""
+    AffineNormalToricVariety(v::NormalToricVariety)
+
+For internal design, we make a strict distinction between normal toric varieties and affine toric varieties. Given a normal toric variety `v` which is affine, this method turns it into an affine toric variety.
```

Comment:
We don't have strict rules on this, but I think it's a good idea in general to wrap lines at somewhere between 80 - 100 columns. If only to make reviews in the GitHub web interface less of a chore (right now this line is only shown partially to me and I have to scroll horizontally a lot to read all of it)

Reply:
I will adjust this.

Yes, I have been making a few more adjustments (to the other doctest failures). It is almost ready, and I hope to push the final version in the next couple of minutes. Will then marked this as "ready for review".